bazel/README.md

```diff
@@ -270,6 +270,16 @@ for how to update or override dependencies.
 1. `bazel build envoy` from the Envoy source directory. Add `-c opt` for an optimized release build or
    `-c dbg` for an unoptimized, fully instrumented debugging build.
 
+## Running Envoy via CLI
+
+After Envoy is built, it can be executed via CLI.
+
+For example, if Envoy was built using the `bazel build -c opt //source/exe:envoy-static` command, then it can be executed from the project's root directory by running:
```

Comment:
If use `ci/do_ci.sh <target>` to build, the output directory is different, please add it as well.

Reply:
Sure!